ChangeLog.d/fix_compilation_ssl_tests.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix an uninitialized variable  warning in test_suite_ssl.function with GCC
+     version 11
```

Comment:
```suggestion
   * Fix an uninitialized variable warning in test_suite_ssl.function with GCC
     version 11.
```
or just
```suggestion
   * Fix an uninitialized variable warning in a test suite with GCC 11.
```
or even no changelog at all. We're inconsistent as to whether we write changelog entries for bug fixes in the tests. I do agree that it makes sense here since this is an issue that users who just want to run the unit tests will face.